charts/clusterpedia/templates/_helpers.tpl

```diff
@@ -306,3 +306,11 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
      {{- .Values.mysql.primary.persistence.selector.matchLabels -}}
 {{- end -}}
 {{- end -}}
+
+{{- define "clusterpedia.storage.password.name" -}}
```

Comment:
Is `clusterpedia.storage.envKeys.password` or `clusterpedia.storage.envs.password` better?

Reply:
Indeed, maybe `clusterpedia.storage.password.envKeys` is better

Reply:
This is good, but `envKey` should be singular.